_search-plugins/neural-sparse-with-raw-vectors.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Using raw vectors
+parent: Neural sparse search
+nav_order: 20
+has_children: false
+---
+
+# Using raw vectors for neural sparse search
+
+If you're using self-hosted sparse embedding models, you can ingest raw sparse vectors and use neural sparse search. 
+
+## Tutorial
+
+This tutorial consists of the following steps:
+
+1. [**Ingest sparse vectors**](#step-1-ingest-sparse-vectors).
+    1. [Create an index](#step-1a-create-an-index).
+    1. [Ingest documents into the index](#step-1b-ingest-documents-into-the-index).
+1. [**Search the data using raw sparse vector**](#step-2-search-the-data-using-a-sparse-vector).
+
+
+## Step 1: Ingest sparse vectors
+
+Once you have generated sparse vector embeddings, you can ingest them into OpenSearch directly.
+
+### Step 1(a): Create an index
+
+In order to ingest documents of raw sparse vectors, create a rank features index:
```

Comment:
Is "of" the right word here, or do we mean something like "containing" or "with"?